demo/index.html

```diff
@@ -221,32 +227,6 @@
       </demo-snippet>
 
       <h3>
-        Tabs can be used in a <code>paper-toolbar</code>.
```

Comment:
Actually, people don't seem to know it exists on iron-selector and which of the required elements inherits from iron-selector